src/operations/execute_operation.ts

```diff
@@ -153,9 +157,14 @@ function executeWithServerSelection(
 
   let selector: ReadPreference | ServerSelector;
 
-  // If operation should try to write to secondary use the custom server selector
-  // otherwise provide the read preference.
-  if (operation.trySecondaryWrite) {
+  if (operation.hasAspect(Aspect.CURSOR_ITERATING)) {
```

Comment:
I called this aspect "cursor iterating" in case there are future commands that behave similar to `getmore`.

Reply:
I called this `CURSOR_ITERATING` since I did not want to confuse it with transaction server pinning, which is not operation specific and thought it might cause confusion.

Reply:
Yea the potential confusion is more than its worth. This is fine.